extensions/amp-bind/0.1/bind-validator.js

```diff
@@ -311,6 +311,14 @@ function createElementRules_() {
       'state': null,
       'is-layout-container': null,
     },
+    'AMP-RENDER': {
+      'src': {
+        'allowedProtocols': {
+          'https': true,
+        },
+      },
+      'state': null,
```

Comment:
FMI: What is this one for? I don't see `state` in the component API design.

Reply:
I misunderstood `state` as `amp-state`. Seems like it was added to support  binding `[localData]` in PR #9376 and seems like we don't support it now. Removed it.